docs/pages/reference/resources.mdx

```diff
@@ -222,3 +226,97 @@ Global configuration options for the Web UI served by the Proxy Service. This re
 Global configuration options for the agents enrolled into automatic updates.
 
 (!docs/pages/includes/cluster-maintenance-config-spec.mdx!)
+
+### Cluster Auth Preferences
+
+Global cluster configuration options for authentication.
+
+```yaml
+metadata:
+  name: cluster-auth-preference
+spec:
+  # Sets the type of second factor to use.
+  # Possible values: "on", "off", "otp", "u2f", "webauthn" and "optional".
```

Comment:
Is u2f still supported? 

cc @codingllama

Reply:
I think you can still write "u2f" as the second_factor mode, but it gets aliased to "webauthn" (and you get a nice warning). Removing it from docs/references seems fine to me.

Reply:
Ok, let's remove it from here then.